plugins/plugin-k8s/src/lib/controller/kubectl.ts

```diff
@@ -75,7 +75,7 @@ interface KubeExecOptions extends ExecOptions {
  */
 const kubelike = /kubectl|oc/
 const isKubeLike = (command: string): boolean => kubelike.test(command)
-
+const sendToPTY = (argv: string[]): boolean => argv.length > 1 && argv[1] === 'exec'
```

Comment:
separate out with whitespace, and add a comment for this function

Reply:
I agree with the new line separation between functions, but why do we need to add a comment? Maybe calling it shouldSendToPTY is enough to understand what it does.